jobclient/pom.xml

```diff
@@ -90,17 +76,18 @@
       <groupId>log4j</groupId>
       <artifactId>log4j</artifactId>
       <version>1.2.17</version>
-  </dependency>
-  <dependency>
+    </dependency>
+    <dependency>
       <groupId>org.jmockit</groupId>
       <artifactId>jmockit</artifactId>
       <version>1.18</version>
```

Comment:
Can we extract this version into a Maven property and also use it in the surefire plugin argument?

Reply:
Done.